compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -322,7 +322,8 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
                     cfg_hide => doc_cfg_hide
                     masked => doc_masked
                     notable_trait => doc_notable_trait
-                    keyword => doc_keyword
+                    keyword => rustdoc_internals
+                    primitive => rustdoc_internals
```

Comment:
I'm pretty sure this will make `doc(primitive)` a hard error. Where is feature-gating of `doc(primitive)` currently handled?

Reply:
It's in `check_attr.rs`.
```
[nix-shell:~/rustc3]$ rg doc_primitive compiler/
compiler/rustc_feature/src/active.rs
664:    (active, doc_primitive, "1.56.0", Some(88070), None),

compiler/rustc_passes/src/check_attr.rs
965:                            if !self.tcx.features().doc_primitive {
```